packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.stories.ts

```diff
@@ -65,3 +91,80 @@ export const WithMessage = {
     message: "Example message",
   },
 } satisfies Story;
+
+/**
+ * This examples shows an empty listbox with default translated message.
+ * You can use the `empty` slot to customize the content.
+ */
+export const Empty = {
+  args: {
+    ...Default.args,
+    options: [],
+  },
+} satisfies Story;
+
+/**
+ * This examples shows a loading listbox.
+ */
+export const Loading = {
+  args: {
+    ...Empty.args,
+    loading: true,
+  },
+} satisfies Story;
+
+/**
+ * This examples shows a loading listbox with lazy loading. The `lazyLoad` event will be emitted if the user scrolls
+ * to the end of the options.
+ */
+export const LazyLoading = {
+  args: {
+    ...Default.args,
+    lazyLoading: {
+      enabled: true,
+    },
+  },
+} satisfies Story;
+
+/**
+ * This examples shows a loading listbox with button loading.
+ */
+export const ButtonLoading = {
+  args: {
+    ...Default.args,
+  },
+  render: (args) => ({
+    setup: () => {
+      return { args, ...useLazyLoading(args.options) };
+    },
+    components: { OnyxListbox, OnyxButton },
+    template: `
+      <OnyxListbox v-bind="args" :options="options">
+        <template #optionsEnd>
+          <OnyxButton label="Load more" mode="plain" :loading="isLazyLoading" style="width: 100%" @click="handleLoadMore" />
```

Comment:
one last mini thing: previously, the load more button had the little "+" icon like in the figma design. it's a mini detail, but Jannick will probably also ask for it in the UX approval, could you add it to the example? :)

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/b4e51082-a475-4f9f-ae85-bef708d15ed6)

and the figma text is a bit different ^^

Reply:
Nice catch! I added it :)